### PR TITLE
fix: OIDC changelog

### PR DESCRIPTION
**Pull request by loujaybee**
## Description

Add OIDC changelog entry. 

## Related Issue(s)

Fixes 

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

- [x] Test all links to make sure they are working correctly
- [x] Verify that all forms (sales, support, docs feedback)  are functioning properly and data is being captured correctly
- [x] Check all buttons and calls-to-action to ensure they are leading to the correct pages
- [x] Ensure that the website is mobile responsive and functions appropriately on different devices
- [x] Please make sure that page is best for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

- [x] Check all website copies for accuracy and spelling/grammar errors
- [x] Verify that all images and videos are displaying properly
- [x] Ensure that all downloadable files are working correctly
- [x] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

- [x] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
- [x] Test website load times to ensure that the website is loading quickly
- [x] Verify that the website can handle high traffic volumes without crashing
- [x] Check that all website functionality is working properly under heavy load
- [x] Perform cross-browser testing to ensure that the website works properly on all major browsers (Google Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

- [x] Ensure that the website meets basic accessibility guidelines.
- [x] Test that users with disabilities can easily navigate and use the website
- [x] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.

- [x] Verify that the website is HTTPS secure and that there are no security vulnerabilities
- [x] Ensure that all sensitive data is being stored and transmitted securely

  </details>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3828"></a>

### CODE REVIEW

**Comment by loujaybee** [comment]
CC: @taliamoyal / @Siddhant-K-code for review 🙏

File: src/content/changelog/2023-07-12-secretless-authorization.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Secretless Authorization (Using OIDC)
+excerpt: with Gitpod you can use OIDC to connect Gitpod workspaces to cloud providers or third parties such as AWS, Azure, GCP, or secret management services like Vault.
+date: 2023-07-12
+image: 2023-07-12-secretless-authorization.jpg
+ogImage: 2023-07-12-secretless-authorization.jpg
+alt: A grey image with a padlock. The header image for changelog announcement for secretless workspaces announcement post.
+---
+
+As of today, with Gitpod you can use OIDC to connect Gitpod workspaces to cloud providers or third parties such as: AWS, Azure, GCP, or secret management services like Vault. Using OIDC integration eliminates the need to manually distribute access credentials, secrets and other key material via other methods such as environment variables. Use `gp idp token` in any workspace (works in `.gitpod.yml` and with prebuilds) to get retrieve the workspace JWT token, to exchange with the OIDC supporting 3rd party for an access token.
```
Comment (loujaybee) [inline src/content/changelog/2023-07-12-secretless-authorization.md:10]:
```suggestion
As of today, with Gitpod you can use OIDC to connect Gitpod workspaces to cloud providers or third parties such as: AWS, Azure, GCP, or secret management services like Vault. Using OIDC integration eliminates the need to manually distribute access credentials, secrets and other key material via other methods such as environment variables. Use `gp idp token` in any workspace (works in `.gitpod.yml` and with prebuilds) to retrieve the workspace JWT token for exchange with the OIDC supporting 3rd party.
```

**Review by Siddhant-K-code — APPROVED** [review]

File: src/content/changelog/2023-07-12-secretless-authorization.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Secretless Authorization (Using OIDC)
+excerpt: With Gitpod you can use OIDC to connect Gitpod workspaces to cloud providers or third parties such as AWS, Azure, GCP, or secret management services like Vault.
+date: 2023-07-12
+image: 2023-07-12-secretless-authorization.webp
+ogImage: 2023-07-12-secretless-authorization.webp
+alt: A grey image with a padlock. The header image for changelog announcement for secretless workspaces announcement post.
+---
+
+With Gitpod you can use OIDC to connect Gitpod workspaces to cloud providers or third parties such as: AWS, Azure, GCP, or secret management services like Vault. Using OIDC integration eliminates the need to manually distribute access credentials, secrets and other key material via other methods such as environment variables.<br/> Use `gp idp token` in any workspace (works in `.gitpod.yml` and with [prebuilds](/docs/configure/projects/prebuilds)) to retrieve the workspace JWT token for exchange with the OIDC supporting 3rd party.
```
Comment (filiptronicek) [inline src/content/changelog/2023-07-12-secretless-authorization.md:10]:
```suggestion
With Gitpod you can use OIDC to connect Gitpod workspaces to cloud providers or third parties such as AWS, Azure, GCP, or secret management services like Vault. Using OIDC integration eliminates the need to manually distribute access credentials, secrets, and other key material via other methods such as environment variables.<br/> Use `gp idp token` in any workspace (works in `.gitpod.yml` and with [prebuilds](/docs/configure/projects/prebuilds)) to retrieve the workspace JWT token for exchange with the OIDC supporting 3rd party.
```

**Review by filiptronicek — APPROVED** [review]

File: src/content/changelog/2023-07-12-secretless-authorization.md (hunk 2 above)
Comment (Siddhant-K-code) [reply src/content/changelog/2023-07-12-secretless-authorization.md:10]: 🫠 missed by a second. I can raise a followup PR

Comment (filiptronicek) [reply src/content/changelog/2023-07-12-secretless-authorization.md:10]: aaah too late 😭

**Pull request merged**